.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Continuous Integration
+
+on:
+    pull_request: ~
+    push: ~
+
+jobs:
+    tests:
+        runs-on: ubuntu-20.04
+        strategy:
+            max-parallel: 1
+            matrix:
+                php:
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                include:
+                    - description: 'Symfony 3.4'
+                      php: '7.2'
+                      composer_option: '--prefer-lowest'
+                    - description: 'Symfony 4.4'
+                      php: '7.3'
+                      symfony: 4.4.*
+                    - description: 'Symfony 5.1'
+                      php: '7.4'
+                      symfony: 5.1.*
+                    - description: 'Symfony 5.2'
+                      php: '8.0'
+                      symfony: 5.2.*
+        name: PHP ${{ matrix.php }} tests (${{ matrix.description }})
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Cache
+              uses: actions/cache@v2
+              with: 
+                  path: ~/.composer/cache/files
+                  key: ${{ matrix.php }}-${{ matrix.symfony }}-${{ matrix.composer_option }} 
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+            - run: |
+                  sed -ri 's/"symfony\/(.+)": "(.+)"/"symfony\/\1": "'${{ matrix.symfony }}'"/' composer.json;
+              if: matrix.symfony
+            - run: composer update --no-interaction --no-progress --ansi ${{ matrix.composer_option }}
+            - run: vendor/bin/simple-phpunit
```

Comment:
`./vendor/bin/simple-phpunit install` should be kept as a separate step so that the output of running tests does not include the installation of PHPUnit as well. running the install separately means it will be collapsed separately (and that separate step can be named `Install PHPUnit` for clarity about what it does)

Reply:
I'm getting problems with that. Installation of simple-phpunit depends on installation of other vendors and on environment variables, that are defined in the matrix.

Reply:
well, I'm asking to put `./vendor/bin/simple-phpunit install` as a separate step just before the one running tests. It can stay after the `composer update` step.